packages/node_modules/@webex/common/src/uuid-utils.js

```diff
@@ -25,6 +25,11 @@ export function constructHydraId(
     throw Error('"type" must be a string');
   }
 
+  if ((type === hydraTypes.PEOPLE) || (type === hydraTypes.ORGANIZATION)) {
+    // Cluster is always "us" for people and orgs
+    return encode(`ciscospark://us/${type.toUpperCase()}/${id}`);
```

Comment:
Does the Hydra team agree with this assumption? I don't know another way of getting the cluster id for a person or org without making a call to the server.

Reply:
I did get @pritomgogoi  to confirm this logic.   I have also anecdotally validated it by decoding all the people and orgIDs that I could find, including ones that I knew belonged to other clusters.